### PR TITLE
build: chart.js and other dependencies

### PR DESCRIPTION
**Pull request by karantakalkar**
## Description
- [x] Install chart.js v3.0.0-alpha

- [x] Uninstall angular-archwizard

- [x] Uninstall swimlane-ngx-charts

- [x] Uninstall d3.js

## Related issues and discussion
#772   

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by karantakalkar** [comment]
@abhaychawla kindly review and merge.

File: package.json [hunk 1]
```diff
@@ -42,10 +42,8 @@
     "@fortawesome/fontawesome-svg-core": "1.2.18",
     "@fortawesome/free-solid-svg-icons": "5.8.2",
     "@ngx-translate/core": "11.0.1",
-    "@swimlane/ngx-charts": "11.1.0",
-    "angular-archwizard": "4.0.0",
+    "chart.js": "2.9.3",
```
Comment (abhaychawla) [inline package.json:45]: The version 3.0 has some breaking changes. Can you check? We can use that from the start to avoid any major change later.

Comment (karantakalkar) [reply package.json:45]: Yes there are https://github.com/chartjs/Chart.js/releases, however it's in pre-release state, should I change version?

Comment (abhaychawla) [reply package.json:45]: Yes, it will prevent any changes related to migration that we might be forced to make at a later stage.

**Comment by abhaychawla** [comment]
Please update the PR description to follow the template as well.

File: package.json (hunk 1 above)
Comment (karantakalkar) [reply package.json]: I will update to full release(after a few months) along with testing utilities.

**Pull request merged**